values.yaml

```
@@ -135,13 +168,20 @@ common-spring-container-stuff: &common-spring-container-stuff
     periodSeconds: 5
 
 common-generic-stuff: &common-generic-stuff
+  template: true
   replicaCount: 1
+  nameOverride: "gs"
```

```suggestion
  nameOverride: "gsc"
```
gs is confusing with GeoSpatial...